pkg/cmd/admin/registry/registry.go

```diff
@@ -95,17 +102,23 @@ const (
 	 * a container and be used on subsequent checks. */
 	healthzRoute               = "/healthz"
 	healthzRouteTimeoutSeconds = 5
+	// this is the official private certificate path on Red Hat distros, and is at least structurally more
+	// correct than ubuntu based distributions which don't distinguish between public and private certs.
+	// Since Origin is CentOS based this is more likely to work.  Ubuntu images should symlink this directory
+	// into /etc/ssl/certs to be compatible.
+	defaultCertificateDir = "/etc/pki/tls/private"
 )
 
 // NewCmdRegistry implements the OpenShift cli registry command
 func NewCmdRegistry(f *clientcmd.Factory, parentName, name string, out io.Writer) *cobra.Command {
 	cfg := &RegistryConfig{
-		ImageTemplate: variable.NewDefaultImageTemplate(),
-
-		Labels:   defaultLabel,
-		Ports:    strconv.Itoa(defaultPort),
-		Volume:   "/registry",
-		Replicas: 1,
+		ImageTemplate:  variable.NewDefaultImageTemplate(),
+		Name:           "registry",
```

Comment:
it is preparing for that changing, currently the command acts the same (name is always overridden in Run based on args)